hknweb/alumni/static/alumni/styles.css

```diff
@@ -0,0 +1,32 @@
+.left-pad {
+    padding-left: 3%;
+}
+
+.detail {
+    padding-left: 5%;
+    line-height: 1.3em;
+    font-size: 1.3em;
+}
+
+.footnote {
+    color: #0000A0;
+    font-size: 0.85em
+}
+
+.blue {
+    color: #0000A0;
+}
+
+#sunset-img {
+    width: 56%;
+    float: left;
+    margin: 0.4%;
+}
+
+#alumni-options {
+    width: 39%;
```

Comment:
Can you explain this 39% to me? Or is this just a number you tested that looks okay?

Reply:
I think so